### DISCUSSION TITLE
kuasor-io/containerd repo should have a tag release

### DISCUSSION THREAD

**Discussion by xiaods** [kuasar-io/kuasar#90] [General]
EXPERIMENTAL in containerd 2.0 milestone: If you desire the full experience of Kuasar, please install [containerd under kuasar-io organization](https://github.com/kuasar-io/kuasar/blob/main/docs/containerd.md). Rest assured that our containerd is built based on the official v1.7.0, so there is no need to worry about missing any functionalities.

use tag to specified commit, it will  suite for go.mod 
developer also will clarity the containerd's update code.

**Comment by Burning1020** [comment]
Thanks. We'll improve it  :)

**Comment by Burning1020** [comment]
We paln to change our baseline to the latest containerd 1.7.x version, and it will take some time.

**Reply by xiaods** [reply]
Thanks
